crates/program/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+use parity_wasm::elements::Instruction;
```

Comment:
can you please add:

``rust
#![deny(missing_docs)]
#![deny(unused)]
#![deny(dead_code)]
#![deny(unreachable_code)]
```